rest-api/src/main/java/com/cloudbees/workflow/rest/external/RunExt.java

```diff
@@ -189,7 +189,10 @@ public void setArtifacts(Link artifacts) {
 
     /** Computes timings after the stages have been set up
      *  That means timing of the stage has been computed, and the stages are sorted
+     *
+     *  Deprecated but retained for external APIs consuming it, use {@link #createNew(WorkflowRun)} instead
```

Comment:
Wait, there is a user-visible bug and the only non-test, non-formatting change is in a method which you are also deprecating? Does not seem to make sense.

Did you instead mean to mark with `@Restricted(NoExternalUse.class)`, i.e., this method is still used in production but was not intended to be called from outside of this plugin and was only accidentally `public`?